antora/docs/modules/ROOT/templates/policy_configuration.hbs

```diff
@@ -212,3 +212,65 @@ interchangeably with the `exclude` field.
 
 Please make the required tooling changes soon to prevent breakages when the
 `non_blocking_checks` field is removed in the future.
+
+== Data Sources
+
+Some of the Enterprise Contract policy rules, defined in the ec-policies git
+repository, rely on certain data values when evaluated. For example, a policy
+rule exists to ensure all the parent container images used during the build
+process have been retrieved from an acceptable registry. The list of acceptable
+registries is a data value. This type of data is called Rule Data.
+
+In addition to policy rules, the ec-policies git repository also defines values
+for Rule Data, see
+link:https://github.com/hacbs-contract/ec-policies/blob/main/data/rule_data.yml[data/rule_data.yml]
+. As a user, you can provide a custom data source with modified Rule Data
+allowing the same policy rules to be used to best fit your use cases.
+
+There are different ways to create a custom data source. The simplest form is to
+fork the ec-policies git repository and change the values of
+`data/rule_data.yml` in your fork. Then, simply provide your fork as a data
+source. For example:
+
+[source,json]
+----
+{
+  "sources": [
+    {
+      "policy": [
+        "git::https://github.com/hacbs-contract/ec-policies.git//policy"
+      ],
+      "data": [
+        "git::https://github.com/lcarva/ec-policies.git//data"
+      ]
+    }
+  ]
+}
+----
+
+It is also possible to provide a conftest bundle as a data source, for example:
```

Comment:
nit: but is it a conftest bundle or opa? The docs say "The format reuses the bundle defined by opa."

Reply:
Good catch. I changed it to OPA bundle but linked to https://www.conftest.dev/sharing/. I didn't want to leave conftest completely out since we do rely on it to build a data only bundle.